README.md

```diff
@@ -8,17 +8,17 @@ See [CONTRIBUTING.md](.github/CONTRIBUTING.md)
 
 ## Developing
 
-This site is built with [Jekyll](https://jekyllrb.com/docs/home/).
+This site is built with [Eleventy](https://www.11ty.dev/).
 
 ### Quick start
 
-This project relies on [Ruby](https://www.ruby-lang.org/en/), [Node](https://nodejs.org/), and [npm](https://www.npmjs.com/). Before following these steps you'll need to [set up a Ruby dev environment](https://jekyllrb.com/docs/installation/) as well as [install node and npm](https://blog.npmjs.org/post/85484771375/how-to-install-npm) if you haven't already.
+This project relies on [Node](https://nodejs.org/), and [npm](https://www.npmjs.com/). Before following these steps you'll need to [install node and npm](https://blog.npmjs.org/post/85484771375/how-to-install-npm) if you haven't already.
+
+Installation:
 
 ```sh
 git clone git@github.com:cloudfour/pwastats.git
 cd pwastats
-gem install bundler
-bundle install
 npm install
 ```
```

Comment:
I know you inherited this, but in trying out the branch locally I think we should revise it to be more of a list format, a la...

```markdown
1. Clone this repository.
1. `cd` into the directory.
1. Run `npm ci` to install dependencies.
```

This avoids the presumption that we're cloning via an SSH path (GitHub defaults to HTTPS, as do I since I use MacOS for auth), and it switches to the faster and more consistent `npm ci` behavior instead of `npm install`.

Reply:
Done